clientconn.go

```diff
@@ -167,6 +173,18 @@ func Dial(target string, opts ...DialOption) (*ClientConn, error) {
 	return cc, nil
 }
 
+func (cc *ClientConn) printf(format string, a ...interface{}) {
```

Comment:
Per chat, can we leave as is? Anything else needed for this PR?
On Aug 25, 2015 5:24 PM, "Qi Zhao" <notifications@github.com> wrote:

> In clientconn.go
> <https://github.com/grpc/grpc-go/pull/287#discussion_r37921644>:
>
> > @@ -167,6 +173,18 @@ func Dial(target string, opts ...DialOption) (*ClientConn, error) {
> >  	return cc, nil
> >  }
> >
> > +func (cc *ClientConn) printf(format string, a ...interface{}) {
>
> rename to eventPrintf and eventErrorf respectively?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/287/files#r37921644>.
>